git_machete/github.py

```diff
@@ -214,6 +214,18 @@ def __fire_github_api_request(method: str, path: str, token: Optional[str], requ
                 f'`{method} {url}` request ended up in 404 response from GitHub. A valid GitHub API token is required.\n'
                 f'Provide a GitHub API token with `repo` access via one of the: {get_github_token_possible_providers()} '
                 'Visit `https://github.com/settings/tokens` to generate a new one.')  # TODO (#164): make dedicated exception here
+        elif err.code == http.HTTPStatus.TEMPORARY_REDIRECT:
+            current_repo_and_org = get_repo_and_org_names_by_id(err.headers['Location'].split('/')[4])
```

Comment:
Just in case... https://github.com/VirtusLab/git-machete/issues/597#issuecomment-1195864082 has an example of lower-case `location` header being returned. Is `err.headers[...]` case-sensitive?

Reply:
@PawelLipski Both lower and upper case first letter works, i just chose upper-case.

Reply:
Okay, probably some case-insensitive `dict` is underneath... pls add a comment in code about that (preferably pointing to an place in the respective `__getitem__` implementation)